test/test_logger.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+require 'sidekiq/logger'
+
+class TesLogger < Minitest::Test
+  describe Sidekiq::Logger do
+    after do
+      Thread.current[:sidekiq_context] = nil
+    end
+
+    describe 'initialization' do
+      describe 'formatter' do
+        subject { Sidekiq::Logger.new(STDERR).formatter }
```

Comment:
Prefer to log to `StringIO` instead of default `STDOUT`/`STDERR`. It's easier to assert content match if needed.

Reply:
Great idea for testing.